Source/OCMockTests/OCMockObjectMacroTests.m

```diff
@@ -363,6 +363,37 @@ - (void)testShouldThrowExceptionWhenNotUsingMockInReject
             NSException, NSInternalInconsistencyException, @"should throw NSInternalInconsistencyException exception");
 }
 
+- (void)testShouldThrowExceptionWhenAttemptingToMockNSObjectMethodInVerify
+{
+  id mock = OCMClassMock([NSString class]);
+
+  XCTAssertThrowsSpecificNamed(OCMVerify([mock description]),
```

Comment:
These should be validating your expected message as well.

Reply:
Done (and updated test above as well to be the same)